src/wp-admin/js/inline-edit-post.js

```diff
@@ -318,7 +390,13 @@ inlineEditPost = {
 
 		return false;
 	},
-
+	/**
+	 * Gets the id for a the post that you want to quick edit from the row in the quick edit table.
+	 * @memberof    inlineEditPost
+	 * @fires       inlineEditPost#getId
+	 * @param       {Object}    o DOM row object to get the id for.
```

Comment:
Strange indentation compared to the other docblocks. Please also check other functions for this.